plugin/src/main/kotlin/com/vanniktech/maven/publish/SonatypeHost.kt

```diff
@@ -8,25 +8,36 @@ import java.io.Serializable
  *
  * https://central.sonatype.org/articles/2021/Feb/23/new-users-on-s01osssonatypeorg/
  */
-data class SonatypeHost(
+data class SonatypeHost internal constructor(
   internal val rootUrl: String,
+  internal val centralPortal: Boolean,
```

Comment:
Maybe?

```suggestion
  internal val isCentralPortal: Boolean,
```

`if (!host.centralPortal) {` reads very weirdly.

Reply:
f842404